addons/cloud_storage_azure/__init__.py

```diff
@@ -0,0 +1,28 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from . import models
+from odoo.exceptions import UserError
+
+
+def uninstall_hook(env):
+    cr = env.cr
+    cr.execute(
+        """
+            SELECT 1
+            FROM ir_attachment
+            WHERE type = 'cloud_storage'
+            AND url LIKE 'https://%.blob.core.windows.net/%'
```

Comment:
Don't you need to escape the `%` characters here?  Suggestion: pass the `LIKE` pattern as a parameter.